test/data/test_env.yaml

```diff
@@ -200,6 +248,12 @@ tool:
               installers:
                 - adoptOpenJdkInstaller:
                     id: jdk-19.0.1+10
+      - name: openjdk-20
+        properties:
+          - installSource:
+              installers:
+                - adoptOpenJdkInstaller:
+                    id: jdk-20.0.1+9
```

Comment:
You should rebase as this is very old commit.

Reply:
I did still showed up. Looks like tests weren't updated for a while.

Reply:
Oh I see, wonder how it is not being detected prior.

Reply:
I believe because that part of the yaml is not being checked / verified by any test